src/download.rs

```diff
@@ -249,11 +249,11 @@ pub fn download_file(
             let downloaded_length = pb.position() - starting_length;
             pb.finish_and_clear();
             let time_taken = starting_time.elapsed();
-            if time_taken != Duration::from_nanos(0) {
+            if !time_taken.is_zero() {
                 eprintln!(
-                    "Done. {} in {} ({}/s)",
+                    "Done. {} in {}s ({}/s)",
                     HumanBytes(downloaded_length),
-                    humantime::format_duration(time_taken),
+                    time_taken.as_secs_f32(),
```

Comment:
I could have used `as_secs_f64()` here, but the resulting output would be too lengthy.

Reply:
To give the output the right length you can format it with `{:.5}` instead of `{}`. HTTPie [does the same](https://github.com/httpie/httpie/blob/c901e704633cb79c686b3edaae497ed38b2978ba/httpie/downloads.py#L32).